assets/src/dashboard/components/popoverMenu/menu.js

```diff
@@ -128,9 +129,9 @@ const Menu = ({ isOpen, currentValueIndex = 0, items, onSelect }) => {
             break;
         }
       };
-
-      document.addEventListener('keydown', handleKeyDown);
-      return () => document.removeEventListener('keydown', handleKeyDown);
+      const listRefEl = listRef.current;
+      listRefEl.addEventListener('keydown', handleKeyDown);
+      return () => listRefEl.removeEventListener('keydown', handleKeyDown);
```

Comment:
this prevents event bubbling to the grid navigation arrow keydowns

Reply:
I haven't tested this, but since we're not attaching to the `document` anymore, could this effect be removed to just use `useCallback` and then pass it as a `onKeyDown` prop to `MenuContainer`?

Reply:
I will try this in the story specific follow up PR i have in draft at the moment!

Reply:
So, updating to use a a callback and attach it to onKeyDown breaks the keyboard control. Because this is a nested focus that uses the same controls as the grid items we're having to really specifically prevent all event bubbling. Even when I preventDefault and stopPropagation on this keydown it's still breaking the flow. 

I don't doubt that there _is_ a way to clean this up, but I've sunk probably half a day into getting the keyboard actions to play nicely within these nested grids. Do you mind if we come back to this later?